framework/src/application/node/processor/processor.ts

```diff
@@ -30,6 +31,10 @@ const forkStatusList = [
 	ForkStatus.DISCARD,
 ];
 
+export const EVENT_PROCESSOR_SYNC_REQUIRED = 'EVENT_PROCESSOR_SYNC_REQUIRED';
```

Comment:
`PROCESSOR_SYNC_REQUIRED`, at places we use the value without `event_`, maybe we should use one format.